certify-mockdp-identity.properties

```diff
@@ -169,5 +169,31 @@ mosip.certify.key-values={\
                                   'background_image': { 'uri': 'https://${mosip.api.public.host}/inji/mosip-logo.png' }, \
                                   'text_color': '#000000'}},\
                     'order' : {'fullName','phone','dateOfBirth','gender','email','region','province','UIN', 'VID', 'postalCode'}\
-                 }}\
+                 },\
+                 "MockVerifiableCredential_mdoc":{\
```

Comment:
Can you remove this from this plugin and add it to `certify-mock-identity.properties` instead?